test/services/psc-verification-link/service.spec.ts

```diff
@@ -86,17 +86,28 @@ describe("PSC Verification Link", () => {
     });
 
     describe("PATCH endpoint", () => {
+        beforeEach(() => {
+            sinon.reset();
+            sinon.restore();
+        });
+
+        afterEach(done => {
+            sinon.reset();
+            sinon.restore();
+            done();
+        });
+
```

Comment:
Can be simplified, see this SO answer and comment: [Cleaning up sinon stubs easily](https://stackoverflow.com/questions/11552991/cleaning-up-sinon-stubs-easily#comment108988900_55251560)

```suggestion
        afterEach(sinon.restore);

```

Reply:
Done for patch and post tests